SwiftWisdomTests/SwiftWisdomTests.swift

```diff
@@ -28,3 +28,30 @@ class GetConstraintTests: XCTestCase {
         XCTAssertNil(constraint, "No constraint there")
     }
 }
+
+class NSMutableAttributedStringTests: XCTestCase {
+    func testAttributedStringApplyingAttributes() {
+        let redText = NSAttributedString(
+            string: "RED",
+            attributes: [NSForegroundColorAttributeName : UIColor.redColor()]
+        )
+        let testString = NSMutableAttributedString(stringWithFormat: "This color is %@", redText)
+        let otherTestString = NSMutableAttributedString(
+            stringWithFormat: "This color is %@",
+            applyingAttributes: [NSForegroundColorAttributeName : UIColor.redColor()],
+            toArgs: "RED"
+        )
+        
+        XCTAssertEqual(testString, otherTestString)
```

Comment:
Does this test the substring colors? ie: if we made one `blueColor`? would it fail?